c/tskit/core.h

```diff
@@ -172,12 +172,12 @@ Used in node flags to indicate that a node is a sample node.
 */
 #define TSK_NODE_IS_SAMPLE 1u
 
-/** 
+/**
```

Comment:
Hmm, I think these lines were mine. Will check editor configuration to try and prevent these.